python/test/unit/language/test_core.py

```diff
@@ -5187,7 +5124,7 @@ def kernel(x):
         pass
 
     x = to_triton(numpy_random((1, ), dtype_str='int32'), device=device)
-    kernel[(1, )](x)
+    kernel.warmup(x, grid=(1, ))
```

Comment:
Deja vu. We probably don't need `test_empty_kernel` and `test_noop`.

Reply:
Sure, just removed both tests.